src/test/CommonVault.t.sol

```diff
@@ -47,7 +47,7 @@ contract CommonVaultTest is TestPlus {
     }
 
     function _getRemainingVaultTvlCap() internal virtual returns (uint256) {
-        return type(uint256).max;
+        return type(uint104).max;
```

Comment:
why are we changing the test here? function still says that its returning uint256 but we are now returning uint104?

Reply:
This is the max amount we can deposit & withdraw. More than that will cause `uint256` overflow.